code/modules/security_levels/keycard_authentication.dm

```diff
@@ -54,7 +54,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/keycard_auth, 26)
 	return data
 
 /obj/machinery/keycard_auth/ui_status(mob/user)
-	if(isdrone(user))
+	if(isdrone(user) || issilicon(user)) //no more borg red alerts
```

Comment:
Why not just exclude borgs as if they don't have access, doing it in ``ui_act`` instead of blocking them from seeing the UI? It's a nothing but just a minor thought.

Reply:
No real reason except that this was the only way I knew how to restrict it - besides, no reason for them to see the UI if they can't access it anyway *shrug*